xpathwebdriver/ColorStreamHandler.py

```diff
@@ -54,44 +61,51 @@ def format(self, record):
         color = self._get_color(record.levelno)
         return color + text + self.DEFAULT
 
+
 class _WinColorStreamHandler(logging.StreamHandler):
     # wincon.h
-    FOREGROUND_BLACK     = 0x0000
-    FOREGROUND_BLUE      = 0x0001
-    FOREGROUND_GREEN     = 0x0002
-    FOREGROUND_CYAN      = 0x0003
-    FOREGROUND_RED       = 0x0004
-    FOREGROUND_MAGENTA   = 0x0005
-    FOREGROUND_YELLOW    = 0x0006
-    FOREGROUND_GREY      = 0x0007
-    FOREGROUND_INTENSITY = 0x0008 # foreground color is intensified.
-    FOREGROUND_WHITE     = FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED
-
-    BACKGROUND_BLACK     = 0x0000
-    BACKGROUND_BLUE      = 0x0010
-    BACKGROUND_GREEN     = 0x0020
-    BACKGROUND_CYAN      = 0x0030
-    BACKGROUND_RED       = 0x0040
-    BACKGROUND_MAGENTA   = 0x0050
-    BACKGROUND_YELLOW    = 0x0060
-    BACKGROUND_GREY      = 0x0070
-    BACKGROUND_INTENSITY = 0x0080 # background color is intensified.
-
-    DEFAULT  = FOREGROUND_WHITE
+    FOREGROUND_BLACK = 0x0000
+    FOREGROUND_BLUE = 0x0001
+    FOREGROUND_GREEN = 0x0002
+    FOREGROUND_CYAN = 0x0003
+    FOREGROUND_RED = 0x0004
+    FOREGROUND_MAGENTA = 0x0005
+    FOREGROUND_YELLOW = 0x0006
+    FOREGROUND_GREY = 0x0007
+    FOREGROUND_INTENSITY = 0x0008  # foreground color is intensified.
+    FOREGROUND_WHITE = FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED
+
+    BACKGROUND_BLACK = 0x0000
+    BACKGROUND_BLUE = 0x0010
+    BACKGROUND_GREEN = 0x0020
+    BACKGROUND_CYAN = 0x0030
+    BACKGROUND_RED = 0x0040
+    BACKGROUND_MAGENTA = 0x0050
+    BACKGROUND_YELLOW = 0x0060
+    BACKGROUND_GREY = 0x0070
+    BACKGROUND_INTENSITY = 0x0080  # background color is intensified.
+
+    DEFAULT = FOREGROUND_WHITE
     CRITICAL = BACKGROUND_YELLOW | FOREGROUND_RED | FOREGROUND_INTENSITY | BACKGROUND_INTENSITY
-    ERROR    = FOREGROUND_RED | FOREGROUND_INTENSITY
-    WARNING  = FOREGROUND_YELLOW | FOREGROUND_INTENSITY
-    INFO     = FOREGROUND_GREEN
-    DEBUG    = FOREGROUND_CYAN
+    ERROR = FOREGROUND_RED | FOREGROUND_INTENSITY
+    WARNING = FOREGROUND_YELLOW | FOREGROUND_INTENSITY
+    INFO = FOREGROUND_GREEN
+    DEBUG = FOREGROUND_CYAN
 
     @classmethod
-    def _get_color(cls, level):
-        if level >= logging.CRITICAL:  return cls.CRITICAL
-        elif level >= logging.ERROR:   return cls.ERROR
-        elif level >= logging.WARNING: return cls.WARNING
-        elif level >= logging.INFO:    return cls.INFO
-        elif level >= logging.DEBUG:   return cls.DEBUG
-        else:                          return cls.DEFAULT
+    def _get_color(cls, level): # Fixme DRY we could encapsulated this login in a top class o external function
```

Comment:
Could we wrap this logic in a parent class or function and inherit it in _AnsiColorStreamHandler, _WinColorStreamHandler?